growthbook.go

```diff
@@ -293,65 +294,63 @@ func (gb *GrowthBook) Run(exp *Experiment) *ExperimentResult {
 }
 
 // Worker function to run an experiment.
-func (gb *GrowthBook) doRun(exp *Experiment) *ExperimentResult {
+func (gb *GrowthBook) doRun(exp *Experiment, featureID *string) *ExperimentResult {
 	// 1. If exp.Variations has fewer than 2 variations, return default
 	//    result.
 	if len(exp.Variations) < 2 {
-		return gb.getExperimentResult(exp, 0, false)
+		return gb.getExperimentResult(exp, -1, false, featureID)
```

Comment:
While this change looks like it changes the logic, I see that `getExperimentResult` will default to 0 when it's less than 0 so this seems fine.